src/test/javascript/controllers/urlInputTest.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Import script
+const script = document.createElement('script');
+script.src = 'src/main/webapp/common_util.js';
+var x = 0;
```

Comment:
You should be able to declare `var i;` here and remove all references to `x`.

Reply:
For some reason, that gives the following error:
* TypeError: Cannot call method "match" of undefined in http://localhost:42095/src/main/webapp/common_util.js (line 26)